source/Textures.js

```diff
@@ -0,0 +1,20 @@
+var Pixi = require("pixi.js")
+
+import Junkership from "./Junkership.js"
+
+exports.initTex = function initTex() {
+    PIXI.loader
+        .add("redJunkership",    require("./images/red-starship.png"))
+        .add("yellowJunkership", require("./images/yellow-starship.png"))
+        .add("greenJunkership",  require("./images/green-starship.png"))
+        .add("blueJunkership",   require("./images/blue-starship.png"))
+        .add("projectile",       require("./images/projectile.png"))
+        .add("trashbot",         require("./images/enemy-starship.png"))
+        .load()
+
+        .on("complete",setup)
+}
```

Comment:
This is neat! What advantages does this have over the old way of loading textures?